src/languages/php.ts

```diff
@@ -0,0 +1,94 @@
+import { SyntaxNode } from "web-tree-sitter";
+import {
+  argumentMatcher,
+  createPatternMatchers,
+  leadingMatcher,
+  trailingMatcher,
+} from "../util/nodeMatchers";
+import {
+  NodeMatcherAlternative,
+  ScopeType,
+  SelectionWithEditor,
+} from "../typings/Types";
+import { getNodeRange } from "../util/nodeSelectors";
+
+// Generated by the following command:
+// > curl https://raw.githubusercontent.com/tree-sitter/tree-sitter-php/0ce134234214427b6aeb2735e93a307881c6cd6f/src/node-types.json \
+//   | jq '[.[] | select(.type == "_statement") | .subtypes[].type]'
+const STATEMENT_TYPES = [
+  "break_statement",
+  "class_declaration",
+  "compound_statement",
+  "const_declaration",
+  "continue_statement",
+  "declare_statement",
+  "do_statement",
+  "echo_statement",
+  "empty_statement",
+  "enum_declaration",
+  "expression_statement",
+  "for_statement",
+  "foreach_statement",
+  "function_definition",
+  "function_static_declaration",
+  "global_declaration",
+  "goto_statement",
+  "if_statement",
+  "interface_declaration",
+  "named_label_statement",
+  "namespace_definition",
+  "namespace_use_declaration",
+  "return_statement",
+  "switch_statement",
+  "trait_declaration",
+  "try_statement",
+  "unset_statement",
+  "while_statement"
+];
+
+export const getTypeNode = (node: SyntaxNode) =>
+  node.children.find((child) => child.type === "type") ?? null;
```

Comment:
What's this function for?

Reply:
Oops I based my file off of the python language just to get started quicker and this was in there, but I don't know what it's for and must have forgot to remove it.